src/install-sfdx/commands/setup.yml

```diff
@@ -0,0 +1,34 @@
+description: |
+  Install `sfdx` CLI and plugins.
+parameters:
+  plugins:
+    description: A comma-delimited list of sfdx plugins to install
+    type: string
+    default: ""
+steps:
+  - restore_cache:
+      keys:
+        - orb-install-sfdx-v1
+  - run:
+      name: Install sfdx CLI + plugins
+      command: |
+        echo "npm install sfdx-cli"
+        npm install sfdx-cli
+        mkdir -p /home/circleci/bin
+        ln -s "$(pwd)/node_modules/sfdx-cli/bin/run" /home/circleci/bin/sfdx
+        mkdir -p $HOME/.config/sfdx
```

Comment:
Would using $HOME everywhere instead of mixing it and /home/circleci (which might be $HOME, not sure) be better?